WordPress/Classes/ViewRelated/Blog/Blog List/BlogListViewController.m

```diff
@@ -1011,18 +981,25 @@ - (void)showAddSiteAlertFrom:(id)source
     if (self.dataSource.allBlogsCount > 0 && self.dataSource.visibleBlogsCount == 0) {
         [self setEditing:YES animated:YES];
     } else {
-        UIAlertController *addSiteAlertController = [self makeAddSiteAlertController];
+        AddSiteAlertFactory *factory = [AddSiteAlertFactory new];
```

Comment:
As discussed via DM, since there are a few ObjC private members accessed in this class (`defaultWordPressComAccount`, `showLoginControllerForAddingSelfHostedSite`, `addSiteAlertController`), I'm leaving this as-is for now.

I do share the thought we should make more of our code Swift, but in this case it's forcing me to do more changes than I'd like to.

I'll try to dig deeper into this idea when implementing the same logic for the blog details, so we can hopefully have shared presentation code.

Reply:
Marking as resolved for now.